arch/SPECS/wazuh-agent/arch/wazuh.install

```diff
@@ -1,8 +1,8 @@
 DIR="PARAM_INSTALLATION_DIR"
 SCRIPTS_DIR="PARAM_INSTALLATION_SCRIPTS_DIR"
 SCA_BASE_DIR="${SCRIPTS_DIR}/sca"
-USER="ossec"
-GROUP="ossec"
+USER="wazuh"
+GROUP="wazuh"
```

Comment:
Does this package remove the ossec user and group in an upgrade process?

You could add installation, uninstallation and upgrade tests to the PR comment to give more visibility on the functionality of the package, you can put the output and the tests that you consider appropriate to show us in dropdowns through 


```
<details>\<summary>title\</summary>

output

</details>
```

Would be see as

<details><summary>title</summary>

output

</details>

Reply:
There is no ossec user, Because this installation script didn't ever work on a release version with the ossec user (The Arch support was only added in version 4.3 when then user was already changed to wazuh. this is why this script never worked).

Reply:
After the package is successfully built (which was not possible until this fix), this is the output you should get:
<details><summary>installation</summary>
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (1) wazuh-agent-4.3.4-1

Total Installed Size:  25.02 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                  [###################################] 100%
(1/1) checking package integrity                                [###################################] 100%
(1/1) loading package files                                     [###################################] 100%
(1/1) checking for file conflicts                               [###################################] 100%
(1/1) checking available disk space                             [###################################] 100%
:: Processing package changes...
(1/1) installing wazuh-agent                                    [###################################] 100%
Creating group 'wazuh' with GID 958.
Creating user 'wazuh' (n/a) with UID 958 and GID 958.
:: Running post-transaction hooks...
(1/3) Creating system user accounts...
(2/3) Reloading system manager configuration...
(3/3) Arming ConditionNeedsUpdate...
</details>
<details><summary>uninstallation</summary>
checking dependencies...

Packages (1) wazuh-agent-4.3.4-1

Total Removed Size:  25.02 MiB

:: Do you want to remove these packages? [Y/n] 
:: Processing package changes...
(1/1) removing wazuh-agent                                      [###################################] 100%
warning: /var/ossec/etc/ossec.conf saved as /var/ossec/etc/ossec.conf.pacsave
:: Running post-transaction hooks...
(1/2) Reloading system manager configuration...
(2/2) Arming ConditionNeedsUpdate...
</details>
<details><summary>update</summary>
loading packages...
warning: wazuh-agent-4.3.4-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Packages (1) wazuh-agent-4.3.4-1

Total Installed Size:  25.02 MiB
Net Upgrade Size:       0.00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                  [###################################] 100%
(1/1) checking package integrity                                [###################################] 100%
(1/1) loading package files                                     [###################################] 100%
(1/1) checking for file conflicts                               [###################################] 100%
(1/1) checking available disk space                             [###################################] 100%
:: Processing package changes...
(1/1) reinstalling wazuh-agent                                  [###################################] 100%
:: Running post-transaction hooks...
(1/3) Creating system user accounts...
(2/3) Reloading system manager configuration...
(3/3) Arming ConditionNeedsUpdate...
</details>